docs/link-expansion.md

```diff
@@ -107,41 +98,31 @@ Within a `links` JSON object there are keys which indicate the type of link
 type. In the above example there are two types of link: `organisations` and
 `available_translations` which contain 1 and 2 links respectively.
 
-## When it occurs
+## Link Lifecycles
 
-Link expansion occurs at the point an edition is represented to the
-Content Store - which is normally the result of a [Sidekiq](http://sidekiq.org/)
-worker process such as [DownstreamDraftWorker][downstream-draft-worker] or
-[DownstreamLiveWorker][downstream-live-worker].
+There are two categories of link with two separate API routes: edition links and link set links.
 
-## How links are added
-
-There are two ways that links can be added for a document: link set links and
-edition links. These are added by different endpoints.
+### `put-content` - Edition links
 
-### `patch-link-set` - Link set links
+These links are added via the [`put-content`](api.md#put-v2contentcontent_id)
+endpoint. They are associated with a single edition of a document.
 
-These links are added via the
-[`patch-link-set`](api.md#patch-v2linkscontent_id) endpoint. They are
-associated with a `content_id`, which therefore associates them with all
-the locales of a document.
+In cases when there are edition links and link set links which have the same
+`link_type`, the edition links will take precedence during link expansion.
 
-They are typically used for taxonomy based links and aren't related to the
-specific content of an edition.
+Edition links should be the default approach to link creation, because link set links have the disadvantage of applying to all editions and locales of a document, which is generally not what is required in most use cases. However, edition links have a substantial limitation, which is that recursive link expansion is not applied to them. If a document needs to access data from documents more than one link away, then link set links must be used.
```

Comment:
Could you link to the section on recursive link expansion here, just in case the reader isn't sure what that is.